docs(ClickUp): add documentation for includeSubtasks flag in get task operation (#3319)
n8n-io/n8n-docs: docs/integrations/builtin/app-nodes/n8n-nodes-base.clickup.md

    * Get all time entry tags
    * Remove tag from time entry

## Operation details

### Get a task

When using the **Get a task** operation, you can enable the **Include Subtasks** option to also fetch and include subtasks for the specified task in the response.

/// note | Include Subtasks parameter

> - **Include Subtasks**: (Boolean, default: false) - When enabled, the node will also fetch and include subtasks for the specified task.

Reviewer: ```suggestion
- **Include Subtasks**: When enabled, also fetches and includes subtasks for the specified task.
```

///

> 

Reviewer: ```suggestion
```

> /// note | Include Markdown Description parameter

Reviewer: ```suggestion
```

> - **Include Markdown Description**: (Boolean, default: false) - When enabled, the node will include the `markdown_description` field in the response, which preserves links and formatting in the task description. This is useful if your task descriptions contain links or rich formatting.

Reviewer: ```suggestion
- **Include Markdown Description**: When enabled, includes the `markdown_description` field in the response, which preserves links and formatting in the task description. This is useful if your task descriptions contain links or rich formatting.
```

> ///

Reviewer: ```suggestion
```
